Makefile

```diff
@@ -1,7 +1,7 @@
 .PHONY: update-version codegen-format
 update-version:
 	@echo "$(VERSION)" > VERSION
-	@perl -pi -e 's|VERSION = "[.\d]+"|VERSION = "$(VERSION)"|' lib/stripe/version.rb
+	@perl -pi -e 's|VERSION = "[.-\w\d]+"|VERSION = "$(VERSION)"|' lib/stripe/version.rb
```

Comment:
There is a backslash in other regexes, not sure if Perl handles it differently

Reply:
Changed for consistency.